ip_repo/amdc_inverters_1.0/README.md

```diff
@@ -0,0 +1,120 @@
+# IP: `amdc_inverters_1.0`
+
+This IP core drives the inverter power stack.
+
+## Features
+
+- 24 Configurable duty ratios (3 for each of the 8 power stack ports)
+- Configurable deadtime
+- Three configurable options for the time at which the duty ratios take effect
+
+## IP Interface
+
+This IP is accessed via the AXI4-Lite register-based interface from the DSP.
+
+## Register Map
+
+| Offset | Name | R/W | Description |
+| -- | -- | -- | -- |
+| 0x00 | DUTY_RATIO_00 | R/W | Configuration Register |
+| 0x04 | DUTY_RATIO_01 | R/W | Configuration Register |
+| 0x08 | DUTY_RATIO_02 | R/W | Configuration Register |
+| 0x0C | DUTY_RATIO_03 | R/W | Configuration Register |
+| 0x10 | DUTY_RATIO_04 | R/W | Configuration Register |
+| 0x14 | DUTY_RATIO_05 | R/W | Configuration Register |
+| 0x18 | DUTY_RATIO_06 | R/W | Configuration Register |
+| 0x1C | DUTY_RATIO_07 | R/W | Configuration Register |
+| 0x20 | DUTY_RATIO_08 | R/W | Configuration Register |
+| 0x24 | DUTY_RATIO_09 | R/W | Configuration Register |
+| 0x28 | DUTY_RATIO_10 | R/W | Configuration Register |
+| 0x2C | DUTY_RATIO_11 | R/W | Configuration Register |
+| 0x30 | DUTY_RATIO_12 | R/W | Configuration Register |
+| 0x34 | DUTY_RATIO_13 | R/W | Configuration Register |
+| 0x38 | DUTY_RATIO_14 | R/W | Configuration Register |
+| 0x3C | DUTY_RATIO_15 | R/W | Configuration Register |
+| 0x40 | DUTY_RATIO_16 | R/W | Configuration Register |
+| 0x44 | DUTY_RATIO_17 | R/W | Configuration Register |
+| 0x48 | DUTY_RATIO_18 | R/W | Configuration Register |
+| 0x4C | DUTY_RATIO_19 | R/W | Configuration Register |
+| 0x50 | DUTY_RATIO_20 | R/W | Configuration Register |
+| 0x54 | DUTY_RATIO_21 | R/W | Configuration Register |
+| 0x58 | DUTY_RATIO_22 | R/W | Configuration Register |
+| 0x5C | DUTY_RATIO_23 | R/W | Configuration Register |
+| 0x60 | CARRIER_CLK_DIV | R/W | Configuration Register |
+| 0x64 | CARRIER_MAX | R/W | Configuration Register |
+| 0x68 | DEADTIME | R/W | Configuration Register |
+| 0x6C | RESETS | R/W | Configuration Register |
+| 0x70 | FLT_TEMP_OUT | R | Status Register |
+| 0x74 | FLT_DESAT_OUT | R | Status Register |
+| 0x78 | RDY_OUT | R | Status Register |
+| 0x7C | CONFIGURATION | R/W | Configuration Register |
+| 0x80 | PER_LEG_ENABLE | R/W | Configuration Register |
+| 0x84 | PER_LEG_REVERSE | R/W | Configuration Register |
+
+### DUTY_RATIO_XX
+
+| Bits | Name | Description |
+| -- | -- | -- |
+| 15:0 | DUTY_RATIO | Duty ratio for a a single inverter leg, expressed as a 16-bit nummber. A value equal to the carrier max implies a duty ratio of 1. |
```

Comment:
a a

Reply:
nummber

Reply:
Man I was really fat-fingering the keys here.